CMakeLists.txt

```diff
@@ -105,6 +105,8 @@ elseif(BASEARCH_S360_FOUND)
     option(WITH_DFLTCC_INFLATE "Build with DFLTCC intrinsics for decompression on IBM Z" OFF)
     option(WITH_CRC32_VX "Build with vectorized CRC32 on IBM Z" ON)
 elseif(BASEARCH_X86_FOUND)
+    option(WITH_AVX512_VNNI "Build with AVX512 VNNI extensions" ON)
```

Comment:
I think you are missing the `add_feature_info` for both `WITH_AVX512` and `WITH_AVX512VNNI` at the bottom of this file.